srcpkgs/python-PyQt5/template

```diff
@@ -3,28 +3,27 @@ pkgname=python-PyQt5
 version=5.13.2
 revision=2
 _sipver=4.19.19
-lib32disabled=yes
 wrksrc="PyQt5-${version}"
-pycompile_module="PyQt5"
 hostmakedepends="pkg-config
- python-devel python3-devel python-sip-devel python3-sip-devel python-dbus-devel
- qt5-tools-devel qt5-connectivity-devel qt5-declarative-devel qt5-location-devel
- qt5-multimedia-devel qt5-sensors-devel qt5-serialport-devel qt5-svg-devel
- qt5-webchannel-devel qt5-webkit-devel qt5-websockets-devel
+ python-devel python3-devel python-sip-devel python3-sip-devel python-dbus-devel qt5
+ qt5-devel qt5-tools-devel qt5-connectivity-devel qt5-declarative-devel qt5-location-devel
+ qt5-multimedia-devel qt5-qmake qt5-sensors-devel qt5-serialport-devel
+ qt5-svg-devel qt5-webchannel-devel qt5-webkit-devel qt5-websockets-devel
  qt5-x11extras-devel qt5-xmlpatterns-devel qt5-networkauth-devel pulseaudio-devel
  python-enum34"
 makedepends="${hostmakedepends/pkg-config/}"
 depends="python-sip-PyQt5>=${_sipver} python-enum34"
 short_desc="Python2 bindings for the Qt5 toolkit"
 maintainer="Alessio Sergi <al3hex@gmail.com>"
-homepage="https://riverbankcomputing.com/software/pyqt/intro"
 license="GPL-3.0-only"
+homepage="https://riverbankcomputing.com/software/pyqt/intro"
 distfiles="https://www.riverbankcomputing.com/static/Downloads/PyQt5/${version}/PyQt5-${version}.tar.gz"
 checksum=adc17c077bf233987b8e43ada87d1e0deca9bd71a13e5fd5fc377482ed69c827
+lib32disabled=yes
 
 pre_build() {
 	mkdir -p pyqt5-${py2_ver}
-	mv * pyqt5-${py2_ver} || true
+	mv * pyqt5-${py2_ver} 2>/dev/null || true
```

Comment:
what is that about? if it fails, why keep it?

Reply:
It fails because `mv: cannot move 'a' to a subdirectory of itself, 'a/a'` (something like that).

This command is necessary but the error looks like it fails, even though in fact it doesn't.

Reply:
It looks like it's trying to move the `pyqt5` folder into itself. Could it be filtered from the glob, perhaps? Using `find` seems like overkill, but should probably work.

Reply:
This change is endup merged into part of #24182 
Actually, I've never like `mv * dst || true`, `create_wrksrc` and make a proper tree instead.

Reply:
`create_wrksrc` is probably actually the correct way to go.